### PR TITLE
Add Apollo Block III+ Service Module & Apollo Block IV Engine & Tidy CFG Format

### PR DESCRIPTION
**Pull request by vader111**
Add Apollo Block III+ Service Module & Apollo Block IV Engine Plus fixes for other issues.

### CODE REVIEW

**Comment by vader111** [comment]
Will Also Need https://github.com/Standecco/ROKerbalism/pull/80

**Comment by vader111** [comment]
(Ready To Review)

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloEngineBlockV.cfg [hunk 1]
```diff
@@ -0,0 +1,171 @@
+PART
+{
+	name = ROC-ApolloEngineBlockV
+	module = Part
+	author = CobaltWolf
+
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloEngineBlockV.cfg:10]: Do not set RP0conf

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloEngineBlockV.cfg [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+		thrustVectorTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalTransformName = thrustTransform
+		gimbalRange = 4.0
+		useGimbalResponseSpeed = True
+		gimbalResponseSpeed = True
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloEngineBlockV.cfg:84]: True is probably not going to work for a speed

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockIII.cfg [hunk 3]
```diff
@@ -76,18 +102,19 @@ PART
...
 	}
-	MODULE
-	{
-		name = ModuleRealAntenna
-		referenceGain = 3.0		//Guess based on Block II
-		RFBand = S
-	}
-}
\ No newline at end of file
+}
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockIII.cfg:120]: The ModuleRealAntenna modules were intentionally duplicated to provide both an omni and a steerable dish.  Is that not correct?

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockV.cfg [hunk 4]
```diff
@@ -5,37 +5,45 @@
...
 	MODULE
 	{
 		name = ModuleRealAntenna
 		antennaDiameter = 1.5	//Guess, no documentation
-	}
-	MODULE
-	{
-		name = ModuleRealAntenna
-		referenceGain = 3.0		//Guess based on Block II
-		RFBand = S
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockV.cfg:39]: The ModuleRealAntenna modules were intentionally duplicated to provide both an omni and a steerable dish. Is that not correct?

**Review by DRVeyl — CHANGES_REQUESTED** [review]

**Review by DRVeyl — APPROVED** [review]

**Comment by DRVeyl** [comment]
I'd appreciate anyone doing some functional testing of these changes before I actually merge.  I'm mostly doing a code review.

**Pull request merged**